spiral/framework
# [Rector] Update to rector 0.12.15, apply parallel, and RemoveUnusedPrivatePropertyRector

| Q             | A
| ------------- | ---
| QA?  | ✔️



- updated rector dependency to 0.12.15
- apply parallel process to speed up check
- apply RemoveUnusedPrivatePropertyRector
- Enable allow-plugins for composer 2.2 support.
